system-parachains/asset-hubs/asset-hub-kusama/src/lib.rs

```diff
@@ -1797,51 +1780,35 @@ cumulus_pallet_parachain_system::register_validate_block! {
 	BlockExecutor = cumulus_pallet_aura_ext::BlockExecutor::<Runtime, Executive>,
 }
 
-#[cfg(feature = "state-trie-version-1")]
 parameter_types! {
 	// The deposit configuration for the singed migration. Specially if you want to allow any signed account to do the migration (see `SignedFilter`, these deposits should be high)
 	pub const MigrationSignedDepositPerItem: Balance = CENTS;
 	pub const MigrationSignedDepositBase: Balance = 2_000 * CENTS;
 	pub const MigrationMaxKeyLen: u32 = 512;
```

Comment:
```suggestion
	pub const MigrationMaxKeyLen: u32 = 32;
```

We have not done this in a parachain, best to do the migration at smaller sizes to be safe.

Please do the same in AH-polkadot too.